src/api/token.ts

```diff
@@ -15,6 +31,28 @@ export class Token {
     this.config = config;
   }
 
+  /**
+   * Create a transaction to mint a token into the creators account within an existing collection.
+   *
+   * @param args.creator the creator of the collection
+   * @param args.collection the name of the collection the token belongs to
+   * @param args.description the description of the token
+   * @param args.name the name of the token
+   * @param args.uri the URI to additional info about the token
+   *
+   * @returns GetTokenDataResponse containing relevant data to the token.
+   */
+  async mintTokenTransaction(args: {
```

Comment:
mintTokenTransaction or mintTransaction?

Reply:
Because they all roll up, they will need Token in the name.